presto-docs/src/main/sphinx/language/types.rst

```diff
@@ -303,6 +303,39 @@ KHyperLogLog
     A KHyperLogLog is a data sketch that can be used to compactly represents the association of two
     columns. See :doc:`/functions/khyperloglog`.
 
+SetDigest
+---------
+
+.. _setdigest_type:
+
+``SetDigest``
+^^^^^^^^^^^^^
+
+A SetDigest (setdigest) is a data sketch structure used
+in calculating `Jaccard similarity coefficient <https://wikipedia.org/wiki/Jaccard_index>`_
+between two sets.
+
+SetDigest encapsulates the following components:
+
+- `HyperLogLog <https://wikipedia.org/wiki/HyperLogLog>`_
+- `MinHash with a single hash function <http://wikipedia.org/wiki/MinHash#Variant_with_a_single_hash_function>`_
+
+HyperLogLog: This is a probabilistic data structure used for the approximation of the number of distinct elements in
+a large dataset. It is based on the principle of observing the 'rarity' of certain patterns in a random process.
+HyperLogLog provides a way to estimate the number of distinct elements (the cardinality) with a high degree
+of accuracy, using a relatively small amount of memory.
+
+MinHash with a single hash function: This is a technique used to quickly estimate the similarity between two sets.
+It involves creating a 'signature' or 'sketch' for each set, which is a much smaller subset that still maintains the
+same properties. The similarity between two sets can then be estimated by comparing their signatures.
+The HyperLogLog structure is used for the approximation of the distinct elements in the original set.
+
+These components work together to provide an efficient and effective way to handle and compare large sets of data.
+
+SetDigests have the property of being additive, which means that you can merge multiple SetDigests together
```

Comment:
```suggestion
SetDigests are additive, so you can merge multiple SetDigests together
```
Does this change alter the meaning of the sentence in a way you feel is less accurate?